src/validation.cpp

```diff
@@ -301,8 +301,8 @@ void Chainstate::MaybeUpdateMempoolForReorg(
     // Iterate disconnectpool in reverse, so that we add transactions
```

Comment:
The comment here about `insertion_order` needs updating.

Reply:
Done, thanks